src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -136,8 +136,8 @@ class NotebookLauncherContent extends Component {
 
     if (url) {
       return iframe({
-        src: url, height: 500,
-        style: { border: 'none' }
+        src: url,
+        style: { border: 'none', flex: 1, marginBottom: '-2rem' }
```

Comment:
Was there not some issue with flex being applied to iframes? If I'm making it up, never mind.

Reply:
I checked in Chrome and FF, seems to work.